charts/jaeger/values.yaml

```diff
@@ -120,6 +120,8 @@ schema:
     #   memory: 128Mi
   serviceAccount:
     create: true
+    # Explicitly mounts the API credentials for the Service Account
+    automountServiceAccountToken: true
```

Comment:
I believe we can make default to `false`
as mounting the serviceAcount token in a Pod is often not necessary. Pods usually don’t need to communicate with the server Api.
Only reason i made it true is to keep backwards compatibility.

Reply:
resolved in comments on the PR.
See https://github.com/jaegertracing/helm-charts/pull/230#issuecomment-812125874